manifests/package/redhat.pp

```diff
@@ -72,14 +72,17 @@
       }
       'passenger': {
         if ($facts['os']['name'] in ['RedHat', 'CentOS', 'VirtuozzoLinux']) and ($facts['os']['release']['major'] in ['6', '7']) {
+          # 2020-11: Passenger changed their gpg key from: `https://packagecloud.io/phusion/passenger/gpgkey`
```

Comment:
(From commit message)
> we see the following error since november 2020

Since 2020-11??

Either way, is it worth splitting this out into its own PR to unblock other contributions?

Reply:
2019 ;)

I added it to this PR because master doesn't build, because our Gemfile pulls in new linter versions.

Reply:
Is there any chance this PR will get merged to address the GPG key issue? Or are we waiting for someone to update it separately?

Reply:
@ardrigh This has been merged separately in https://github.com/voxpupuli/puppet-nginx/pull/1408